modules/client-handler/src/main/java/org/apache/ignite/client/handler/requests/sql/ClientSqlExecuteScriptRequest.java

```diff
@@ -17,17 +17,15 @@
 
 package org.apache.ignite.client.handler.requests.sql;
 
-import static org.apache.ignite.client.handler.requests.sql.ClientSqlExecuteRequest.readSessionProperties;
+import static org.apache.ignite.client.handler.requests.sql.ClientSqlExecuteRequest.readSession;
```

Comment:
Looks a bit weird to me. Let's move it to `ClientUtils`, or something.

Reply:
Moved to `ClientSqlCommon`.